proposal/q40603/proposal.md

```diff
@@ -0,0 +1,153 @@
+## Proposal for Pairs Trading 
+
+
+A pairs trade is a trading strategy that involves matching a long position with a short position in two stocks with a high correlation. It uses statistical and technical analysis to seek out potential market-neutral profits.
+
+In this term project, I would like to optmize the Pairs Trading computing engine developed by the classmate in my lab. He combined the pairs trading techinque with Taiwan Stock Market intraday high-frequency trading, whcih I think is an opportunity and is suitalbe for NSD term project.
+
+### Basic information
+
+Github repo : [https://github.com/q40603/pairs_trade_nsd](https://github.com/q40603/pairs_trade_nsd)
+
+Pairs Trading always comes along with a term "Cointegration". The basic ideas of Cointegration is to test whether two time series move similarly. When applied to stock market, we can find two assets that move with cointegration, assign different weight at the assets respectively, and sell the high priced stock, buy the low priced stock. 
+
+| two stock series | Coinegration series |
+| -------- | -------- |
+| ![](https://i.imgur.com/ajKsdUc.png)     | ![](https://i.imgur.com/2cPbJ7F.png)    |
+
+
+By using this strategy, we can gain the profit in both bear market and bull market. However, trading the assets too frequesntly may suffers from the huge trading cost.  
+
+
+
+
+
+
+
+
+### Problem to solve
+
+
+#### Prerequiement
+
+* For each trading day ( 交易日 ), we can trade the stock from 9:00 ~ 13:30
+* To test whether two stock price has cointegration, we must collect sufficient data to do cointegration test.
+* We choose the top 150 company stock price as our input data.
+* Some of the stock suffers from bad liquidity, which means there exists empty value in our data. So we compute the average price within each minute, and backfill the data with empty value.
+* We collect the data between 9:16 ~ 11:46 to do cointegration test
+* Trade the stock from 11:46~13:30
+
+The original version of the Pairs Trading computing engine is written in Python, and it takes about 2 minutes to do the computing:
+* We have to test every two stock of top 150 company (11175 combinations)
+
+#### Math Part
+Consider a K-dimensional  Gaussian VAR(p) model with a trend component:
+
+![](https://i.imgur.com/gfkwosJ.png)
```

Comment:
The equations will be good to be included in a written report or presentation slides.  If you have not done it, you are encouraged to do it.